### PR TITLE
Support for WhatsApp Web 2.2241.6

### LINKED ISSUE #1758: Cannot read properties of undefined (reading 'isMDBackend')
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Cannot read properties of undefined (reading 'isMDBackend')

### Expected behavior

to function well

### Steps to Reproduce the Bug or Issue

just run a normal cose 

### Relevant Code

_No response_

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

latest versions 

### Additional context

no more

### PR DESCRIPTION
**Pull request by purpshell**
# PR Details

Fixes an issue where isMDBackend seems to be removed

close #1758

## How Has This Been Tested

I tested it, please test it too

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Comment by andresmv94** [comment]
the complete solution that works for me is this one : [https://github.com/pedroslopez/whatsapp-web.js/issues/1758#issuecomment-1292418580](https://github.com/pedroslopez/whatsapp-web.js/issues/1758#issuecomment-1292418580)

**Comment by PurpShell** [comment]
@pedroslopez @Aliyss @tuyuribr ready for review and approval

**Comment by PurpShell** [comment]
> the complete solution that works for me is this one : [#1758 (comment)](https://github.com/pedroslopez/whatsapp-web.js/issues/1758#issuecomment-1292418580)

Fixed

**Review by tuyuribr — APPROVED** [review]

**Review by pedroslopez — APPROVED** [review]

File: src/util/Injected.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ exports.ExposeStore = (moduleRaidStr) => {
     window.Store.Cmd = window.mR.findModule('Cmd')[0].Cmd;
     window.Store.CryptoLib = window.mR.findModule('decryptE2EMedia')[0];
     window.Store.DownloadManager = window.mR.findModule('downloadManager')[0].downloadManager;
-    window.Store.MDBackend = window.mR.findModule('isMDBackend')[0].isMDBackend();
+    if (window.mR.findModule('isMDBackend')[0]?.isMDBackend) {
```
Comment (pedroslopez) [inline src/util/Injected.js:16]: Missed this - we can't use `?.` since we still support Node v12, which doesn't have this.

**Review by pedroslopez — CHANGES_REQUESTED** [review]

File: src/util/Injected.js (hunk 1 above)
Comment (crawer) [reply src/util/Injected.js:16]: Got a problem here.

**Comment by pedroslopez** [comment]
Superseded by #1762

**Pull request closed without merging**